packages/vx-demo/src/components/Gallery.tsx

```diff
@@ -63,24 +63,31 @@ const items = [
 ];
 
 const tiltOptions = { max: 8, scale: 1 };
+const detailsHeight = 76;
 
 export default function() {
-  const detailsHeight = 76;
   return (
     <div>
       <div className="gallery">
         <Tilt className="tilt" options={tiltOptions}>
-          <Link href="/lines">
-            <div className="gallery-item" style={{ background: items[0] }}>
+          <Link href="/curves">
+            <div className="gallery-item" style={{ border: '1px solid lightgray' }}>
```

Comment:
extract constant outside?

Reply:
yeah, the gallery page currently sets a lot of ad hoc styles. I may address in another PR bc it's a lot.

Reply:
sure. np.